packages/frameworks-sveltekit/src/lib/index.ts

```diff
@@ -270,28 +270,38 @@ export function SvelteKitAuth(
   return async function ({ event, resolve }) {
     const _config = typeof config === "object" ? config : await config(event)
     setEnvDefaults(env, _config)
-    const { prefix = `${base}/auth` } = _config
+    _config.basePath ??= `${base}/auth`
 
     const { url, request } = event
 
-    event.locals.getSession ??= () => getSession(request, _config)
+    event.locals.auth ??= () => auth(event, _config)
+    event.locals.getSession ??= event.locals.auth
 
     const action = url.pathname
-      .slice(prefix.length + 1)
+      .slice(_config.basePath.length + 1)
       .split("/")[0] as AuthAction
```

Comment:
```suggestion
      .split("/")[0]
```

We don't know the type here yet